packages/caliper-vscode/extension/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
```

Comment:
The changelog is currently maintained at the repo level, so this file is not needed.